### PR TITLE
[CHORE] App package updates

### PR DESCRIPTION
**Pull request by benjaminpkane**
## What changes are proposed in this pull request?

Package update audit

## How is this patch tested? If it is not, please explain why.

e2e, unit tests

### What areas of FiftyOne does this PR affect?

-   [x] App: FiftyOne application changes
-   [x] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [ ] Documentation: FiftyOne documentation changes
-   [ ] Other

### CODE REVIEW

**Comment by benjaminpkane** [comment]
@coderabbitai review

File: app/packages/app/package.json
[the remark this answers is not in the document]
Comment (benjaminpkane) [reply app/packages/app/package.json:1]: They are declared in `@fiftyone/core`. The `@fiftyone/app` package does not require them. An artifact of the routing work

**Review by swheaton — APPROVED** [review]
ok lgtm

**Comment by findtopher** [comment]
depcheck says:
```
❯ depcheck app
Unused dependencies
* jpeg-js
Unused devDependencies
* @testing-library/react
* @testing-library/react-hooks
* @vitest/coverage-v8
* typedoc
* typescript
* typescript-plugin-css-modules
* vite-plugin-eslint
Missing dependencies
* lodash: ./gen-docs.js
* recoil: ./__mocks__/recoil-relay.ts
* recoil-relay: ./__mocks__/recoil-relay.ts
❯ git rev-parse --short HEAD
fb9971f60
```

For what that's worth...

`make app` on `develop` takes 2m 16s
`make app` on `chore/app-package-updates` (after cleaning `node_modules` directories and npm cache) takes 2m 14s

So it all seems pretty much the same to me!

**Pull request merged**